Engine/Audio/Adapter/SDLAudioEngineAdapter.cpp

```diff
@@ -2,12 +2,126 @@
 // Created by Storm on 05/10/2020.
 //
 
+#include <vector>
 #include "SDLAudioEngineAdapter.hpp"
 
-void SDLAudioEngineAdapter::getAudioFiles() {
+SDLAudioEngineAdapter::SDLAudioEngineAdapter() {
 
+    _lastPlayedMusic = Mix_LoadMUS("kda.vav");
+
+    // TODO: Try higher frequency
```

Comment:
TODO al opgelost? Anders even een ticket voor aanmaken en het nummer van de ticket vermelden in de todo comment.